src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,20 +56,39 @@
   // private final XboxController m_controller = new XboxController(0);
   private Joystick joystick1;
   private Joystick joystick2;
+  private Joystick joystick3;
+
+    // joystick1 buttons
+  private JoystickButton driverIntakeButton;
+  private JoystickButton driverFeedButton;
+  private JoystickButton driverEjectButton;
+  private JoystickButton driverStartShootin;
```

Comment:
isn't this a joystick2 button?

Reply:
it is I just forgot to push my changes that assigned all these to the correct buttons, so it will be coming with the rest of the corrections from this review